client/src/components/HeightClipper/HeightClipper.tsx

```diff
@@ -37,10 +42,39 @@ export class HeightClipper extends React.Component<
     };
     this.main = React.createRef();
     this.content = React.createRef();
+
+    this.debounced_mutation_callback = _.debounce(
+      (mutationList: MutationRecord[], observer: MutationObserver) => {
+        const height_clipper_node = ReactDOM.findDOMNode(this) as HTMLElement;
+        const untabbable_children_node = height_clipper_node.querySelector(
+          ".untabbable_children"
+        );
```

Comment:
Could just refs for this (thought that's what the existing ones were for, but haven't looked at the rest of this component too closely).

Reply:
I might've misunderstood your comment, apologies if the new push doesn't address this comment.

Reply:
No worries. Feel free to ask for clarification any time (or take a stab at it and put things back up for review, up to you case-by-case).